test/__spanner-test/spanner-test.ts

```diff
@@ -0,0 +1,81 @@
+// import { Spanner } from "@google-cloud/spanner"
```

Comment:
Should this file be commented out?

Reply:
@AlexMesser

Reply:
this folder was removed in one of subsequent PRs